File: app/src/main/java/com/aa/pacer/SettingsFragment.java
package com.aa.pacer;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.preference.PreferenceFragmentCompat;

public class SettingsFragment extends PreferenceFragmentCompat
{
    @Override
    public void onCreate(Bundle savedInstanceState) 
    {
        super.onCreate(savedInstanceState);

        setPreferencesFromResource(R.xml.prefs, null);
    }

    @Override
    public void onCreatePreferences(@Nullable Bundle savedInstanceState, @Nullable String rootKey) {

    }
}

File: app/src/main/java/com/aa/pacer/SettingsFragment.kt
package com.aa.pacer

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat

class SettingsFragment : PreferenceFragmentCompat() {

    private val RINGTONE_NAME = "ringtoneName"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setPreferencesFromResource(R.xml.prefs, null)
    }
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {}
}